instances/technique/widefieldMicroscopy.jsonld

```diff
@@ -6,9 +6,9 @@
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
   "definition": "'Widefield microscopy' comprises all microscopy techniques in which the entire sample is permanently exposed to a light source.",
```

Comment:
Maybe remove the ' around widefield microscopy for consistency purposes (since they are not used in any of the other definitions? - "Widefiled microscopy comrpises all microscopy ..."